### ISSUE TITLE
UI: Show what version is being run 

### ISSUE THREAD

**Issue by exitus1** [decred/dcrdex#3091]
People upgrading are asking how to know if they upgraded. Would be nice to have the version # somewhere in settings.

https://x.com/julianchuk/status/1859401033149014027

**Comment by norwnd** [comment]
For additional context,

we have [Build ID](https://github.com/decred/dcrdex/issues/3088) (in settings) already, but it would also be nice to additionally have a "release version" (in case a commit corresponds to official release) - since it removes 1 additional step of linking Build ID to release **manually**.

**Comment by exitus1** [comment]
Looks good. Maybe call it "Version" instead of About?

**Comment by dev-warrior777** [comment]
Can, but then I would just put the version at the bottom of the settings dialog. The build ID is problematic as it may not be available or may be incorrect as can be seen by the long conversation in the dex-dev channel with jrick/davec.

Ok, first I will look at making sure we have at least the version at the bottom of the  Settings dialog and _no_ separate About dialog .. which is _very_ overkill just for Version. Then if there is a valid git commit (based on what the go code says) .. put that underneath.

Then users can just hit the settings link to get the version info.

**Comment by norwnd** [comment]
> Can, but then I would just put the version at the bottom of the settings dialog.

Build ID is a sort of version, just a different one. I think what @exitus1 suggests is a cleaner way to go about it, but given both version and trimmed-commit are short enough it might be possible to just combine them together into something like `v1.0.2 (1fd2fce)` to put at the bottom of the settings dropdown menu.

**Comment by dev-warrior777** [comment]
>@dev-warrior777 while you are modifying this dropdown menu do you think you could also address these ?
 
I will not be changing the drop-down .. rather arranging for app.Version to be visible in the Settings dialog underneath the Redeem game code div/section. With or without the problematic Build ID.
I can look at these but not on the PR coming out of this. @martonp might look at `Market Making` link versus Market Making robot icon click

Did you read @jrick 's helpful comments in dex-dev channel?

**Comment by norwnd** [comment]
> Did you read @jrick 's helpful comments in dex-dev channel?

Most of it, why ? If you mean we can't always have the version available - I guess just set it to `unknown` as fallback, but that's probably a rare case to hit in practice (most users will use pre-built binary, and obviously the one releasing those binaries should build it such that both version and commit is available).

**Comment by dev-warrior777** [comment]
No .. we can show the actual build always I think. PR later today.

**Comment by dev-warrior777** [comment]
> "Market Making" tab in there seems to be redundant since "MM bot" button (see screenshot below) seems to already lead user to market-making page

Yes .. both go to `host/mm` .. maybe one can be removed .. let's see what @martonp wants to do

**Comment by dev-warrior777** [comment]
> People upgrading are asking how to know if they upgraded. Would be nice to have the version # somewhere in settings.

Workaround for now if the person is comfortable with console/command prompt/dos box then find the executable such as `bisonw.exe`, `bisonw` .. open a command prompt and:
- `C: bisonw.exe --version`
- `$  ./bisonw --version`

Should show `bisonw version 1.0.2+release.local (Go version go1.23.0 linux/amd64)` with latest release
